docs/how-to-run-simulated.adoc

```diff
@@ -30,6 +30,13 @@ Prerequisite software may be installed with the following script:
 $ ./tools/install_builder_prerequisites.sh
 ----
 
+Now put these on your PATH with:
+
+[source,text]
+----
+$ source env.sh
+----
+
```

Comment:
We might want to advise running this _before_ installing prerequisites, as I think the script will bail out early if the path isn't already set up?